core/src/main/java/org/frankframework/management/bus/endpoints/WebServices.java

```diff
@@ -245,20 +246,28 @@ public List<ListenerDAO> getRestListeners() {
 	@JsonInclude(Include.NON_NULL)
 	public class ListenerDAO {
 		private final @Getter String name;
-		private final @Getter String method;
+		private final @Getter List<HttpMethod> method;
 		private final @Getter String uriPattern;
 		private @Getter String receiver;
 		private @Getter String adapter;
 
 		public ListenerDAO(RestListener listener) {
 			this.name = listener.getName();
-			this.method = listener.getMethod();
+
+			HttpMethod tempMethod;
+			try {
+				tempMethod = HttpMethod.valueOf(listener.getMethod().toUpperCase());
+			} catch (IllegalArgumentException e) {
+				tempMethod = HttpMethod.GET; // Default to GET, when parsing of enum fails
+			}
```

Comment:
Is catching the best way here, or should we throw a BusException, or put no method as default?

Reply:
Je kan het beste EnumUtils.parse gebruiken, dan maakt de casing niet uit. Wat betreft de illegalArgument denk ik dat je beter een log.warn kan geven en de waarde null kan/moet geven.